test/e2e/traceflow_test.go

```diff
@@ -320,3 +333,29 @@ func (data *TestData) createNPAllowAllEgress(name string) (*networkingv1.Network
 	}
 	return data.createNetworkPolicy(name, spec)
 }
+
+// waitForNetworkpolicyRealized waits for the NetworkPolicy on the pod to be realized.
+func (data *TestData) waitForNetworkpolicyRealized(pod string, networkpolicy string) error {
+	if err := wait.Poll(1*time.Second, defaultTimeout, func() (bool, error) {
+		cmds := []string{"antctl", "get", "networkpolicy", "-o", "json", "-n", testNamespace}
```

Comment:
you can specify name directly, exit code won't be 0 if not found, so don't have to parse the result.
```
  Get a specific NetworkPolicy
  $ antctl get networkpolicy np1 -n ns1
```

Reply:
Updated.